content/en/docs/setup/additional-setup/customize-installation-helm/index.md

```diff
@@ -0,0 +1,147 @@
+
+First, we create a `sysctl` patch file, adding a `securityContext` to the `ingress-gateway` pod with the additional attribute:
+
+{{< text bash >}}
+$ cat > sysctl-ingress-gw-customization.yaml <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ingress-gateway
+spec:
+  template:
+    spec:
+      securityContext:
+          sysctls:
+          - name: net.netfilter.nf_conntrack_tcp_timeout_close_wait
+            value: "10"
+EOF
+{{< /text >}}
+
+The below shell script helps to bridge the gap between Helm post renderer and Kustomize, as the former works with `stdin/stdout`
+and latter works with files.
+
+{{< text bash >}}
+$ cat > kustomize.sh <<EOF
+#!/bin/sh
+cat > base.yaml
+exec kubectl kustomize # you can also use "kustomize build ." if you have it installed.
+EOF
+chmod +x ./kustomize.sh
+{{< /text >}}
+
+Finally, let us create the `kustomization` yaml file, which is the input for `kustomize`
+with the set of resources and associated customization details.
+
+{{< text bash >}}
+$ cat > kustomization.yaml <<EOF
+resources:
+- base.yaml
+patchesStrategicMerge:
+- sysctl-ingress-gw-customization.yaml
+EOF
+{{< /text >}}
+
+## Apply and Verify the Customization
+
+Now that the customization file is ready, let us use Helm to make sure this gets applied properly.
+
+### Add the Helm repository for Istio
+
+{{< text bash >}}
+$ helm repo add istio https://istio-release.storage.googleapis.com/charts
+$ helm repo update
+{{< /text >}}
+
+### Render and Verify using Helm Template
+
+We can use Helm `post-renderer` to validate rendered manifests before they are installed by Helm
+
+{{< text bash >}}
+$ helm template istio/gateway --post-renderer ./kustomize.sh
+{{< /text >}}
+
+In the output, check for the newly added `sysctl` attribute for `ingress-gateway` pod:
+
+{{< text yaml >}}
+        - mountPath: /etc/envoy
+          name: envoy-config
+      securityContext:
+        sysctls:
+        - name: net.netfilter.nf_conntrack_tcp_timeout_close_wait
+          value: "10"
+{{< /text >}}
+
+### Apply the patch using Helm Post Renderer
+
+Use the below command to install an Istio ingress-gateway, applying our customization using Helm `post-renderer`:
+
+{{< text bash >}}
+$ kubectl create ns istio-ingress
+$ helm upgrade -i istio-ingress istio/gateway --namespace istio-ingress --wait --post-renderer ./kustomize.sh
+{{< /text >}}
+
+Examine the ingress-gateway deployment, you will see the newly manipulated `sysctl` value:
+
+{{< text yaml >}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  …
+  name: istio-ingress
+  namespace: istio-ingress
+spec:
+  template:
+    metadata:
+      …
+    spec:
+      securityContext:
+        sysctls:
+        - name: net.netfilter.nf_conntrack_tcp_timeout_close_wait
+          value: "10"
+{{< /text >}}
+
+## References
+
+1. [Gloo Edge - Last Mile Helm Chart Customization](https://docs.solo.io/gloo-edge/latest/installation/gateway/kubernetes/helm_advanced/)
```

Comment:
I don't think we need to include this; users shouldn't need this with the doc we have

Thanks to the Solo team for the great doc though!

Reply:
Updated